hack/jenkins/job-configs/test-linkchecker.yaml

```diff
@@ -0,0 +1,66 @@
+- job-template:
+    name: 'kubernetes-test-linkchecker'
+    description: |
+        Grab the latest from GitHub, then run hack/verify-linkcheck.sh.<br>
+        Test Owner: Build Cop
+    logrotate:
+        numToKeep: 200
+    builders:
+        - shell: './hack/verify-linkcheck.sh'
+    publishers:
+        - claim-build
+        - gcs-uploader
+        - log-parser
+        - email-ext
+        - xunit:
```

Comment:
you probably don't want this xunit block, since `hack/verify-linkcheck.sh` doesn't produce any JUnit output AFAIK.